apps/vx-scan/backend/src/app.ts

```diff
@@ -129,63 +142,6 @@ export function buildApp(
   );
   deprecatedApiRouter.use(express.urlencoded({ extended: false }));
 
-  deprecatedApiRouter.patch<
-    NoParams,
-    Scan.PatchMarkThresholdOverridesConfigResponse,
-    Scan.PatchMarkThresholdOverridesConfigRequest
-  >('/precinct-scanner/config/markThresholdOverrides', (request, response) => {
-    const bodyParseResult = safeParse(
-      Scan.PatchMarkThresholdOverridesConfigRequestSchema,
-      request.body
-    );
-
-    if (bodyParseResult.isErr()) {
-      const error = bodyParseResult.err();
-      response.status(400).json({
-        status: 'error',
-        errors: [{ type: error.name, message: error.message }],
-      });
-      return;
-    }
```

Comment:
Love all this parsing logic that we don't have to worry about anymore!

Reply:
It was v satisfying to delete it